### ISSUE TITLE
MSFS World Update 16 Sangster Intl (MKJS) Issue

### ISSUE THREAD

**Issue by Viperstrike95** [albar965/littlenavmap#1085]
Hey albar, I wanted to raise up a bug or issue that arose after the new World Update that came out on Jan 30, 2024. When most of the airports that were handcrafted got updated, the procedures were unchanged. However for Sangster Intl (MKJS) got the handcrafted scenery, the procedures list no longer shows up unless you turn on the Navigraph setting (I don't use Navigraph for Little NavMap due to me not having a subscription). I bet if I uninstall the scenery and update the scenery library, the procedures should return. I would like to keep the scenery. I'm hoping that gets fixed soon.

**Comment by Viperstrike95** [comment]
Here's what is on my screen, it is the handcrafted scenery for MKJS that came with the new World Update. And after downloading the new World Update and updating the scenery library, the procedures that were there from LNM are no longer there.

![image](https://github.com/albar965/littlenavmap/assets/158427890/2096bd62-6d25-48d9-8c8e-5a72ae53fa55)

![image_2024-02-01_134236237](https://github.com/albar965/littlenavmap/assets/158427890/5f8f6de2-035e-4d6b-808b-dc81c8ddd179)

**Comment by albar965** [comment]
Thank you for the screen shots.
Can you attach the file 'MKJS_Scene.bgl' here (or send it per email https://albar965.github.io/contact.html )?
You can jump directly to it by clicking the blue link in 'Data Sources'.

Not sure if I can fix this. Might be that MS messed up again.

Alex

**Comment by albar965** [comment]
Sorry, no file. Login into Github and attach it here because their email interface is messy.

**Comment by Viperstrike95** [comment]
Just tried and it says that Github doesn't support the file type.

**Comment by albar965** [comment]
Send per email then. Contact link above.

**Comment by Viperstrike95** [comment]
Just sent the email to you.

**Comment by Viperstrike95** [comment]
Cool. I'm hoping that you could resolve the issue at some point. If not, I'll just the navigraph setting to get the procedures. Thanks again for your help and I'll let you know if I have any more issues.

**Comment by albar965** [comment]
BTW: Do you use GSX or other add-ons that modify airports as well? Try to exclude them from loading in LNM options on page 'Scenery Library Database`.

I know a few cases where these caused issues.

**Comment by Viperstrike95** [comment]
I have never used GSX or any other related add-ons in MSFS at all (or even purchased/downloaded them).

**Comment by albar965** [comment]
I can reproduce it. 

The BGL instructs the simulator to replace the procedures (SID, STAR and approaches) of the stock airport with the ones in the BGL but the add-on BGL does not contain anything to replace the stock procedures. Oh dear. Microsoft. :roll_eyes: 

Trying a workaround. 

I suppose the airport shows procedures in MSFS, right?

**Comment by Viperstrike95** [comment]
Yes it does for both the PMDG 737 (Which has Navigraph already in the plane) and in the default Cessna Citation CJ4 (Which uses the data in the sim itself).

**Comment by albar965** [comment]
Ok. Thanks.

**Comment by albar965** [comment]
I added a workaround for the next LNM 3.0.2.beta. Hopefully in a few days or I can send you a development release per email.
2.8.12  will no longer receive updates.
